tests/unit/ray_tracing.cpp

```diff
@@ -1620,255 +1620,254 @@ TEST_F(NegativeRayTracing, CmdBuildAccelerationStructuresKHR) {
     }
 
     // dstAccelerationStructure == VK_NULL_HANDLE
-    {  // Command buffer build
-        {
-            auto blas = vkt::as::blueprint::BuildGeometryInfoSimpleOnDeviceBottomLevel(*m_device);
-            blas.SetDstAS(vkt::as::blueprint::AccelStructNull(*m_device));
-
-            m_errorMonitor->SetDesiredError("VUID-vkCmdBuildAccelerationStructuresKHR-dstAccelerationStructure-03800");
-            blas.BuildCmdBuffer(m_command_buffer);
-            m_errorMonitor->VerifyFound();
-        }
-    }
+    {// Command buffer build
+     {auto blas = vkt::as::blueprint::BuildGeometryInfoSimpleOnDeviceBottomLevel(*m_device);
```

Comment:
```suggestion
     auto blas = vkt::as::blueprint::BuildGeometryInfoSimpleOnDeviceBottomLevel(*m_device);
```

this is causing Clang-format to act up here in this test